README.md

```diff
@@ -0,0 +1,381 @@
+The important info is your Public DNS / Public IP adress.
+
+
+![AWS_13.png](/docs/AWS_13.png)
+
+You can ssh to your instance with the user "ubuntu", see [here](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/managing-users.html) for more info on
+user accounts on linux instances. In terminal window run this command with your instance infos:
+
+```
+ssh -l ubuntu -i YOUR-PEM-FILENAME.pem YOUR-AWS-EC2-PUBLIC-DNS
+```
+
+Now you should be ready with your instance and see the welcome information of ubuntu.
+
+
+
+As next step go to back to the [Quickstart](#quickStart) and follow the installation steps.
+
+
+
+Next we have to configure nginx as webserver and copy the openHAB Cloud nginx config, overriding the default config:
+```
+sudo cp /home/ubuntu/openhabcloud/etc/nginx_openhabcloud.conf /etc/nginx/sites-available/default
+```
+
+Change the following lines to match your instance.
+Point `server_name` to your AWS Public DNS
+
+
+```
+cd /etc/nginx/sites-enabled
+sudo vi default
+```
+
+```
+server {
+#listen *:443;
+listen *:80;
+#ssl on;
+# ssl_certificate /etc/nginx/ssl/YOUR-KEY.crt;
+# ssl_certificate_key /etc/nginx/ssl/YOUR-KEY.key;
+#ssl_certificate /etc/nginx/ssl/YOUR-CER.crt;
+#ssl_certificate_key /etc/nginx/ssl/YOUR-KEY.key;
+
+server_name YOUR-AWS-EC2-PUBLIC-DNS;
+
+#if ( $scheme = "http" ) {
+#    rewrite ^/(.*)$    https://$host/$1 permanent;
+#}
+
+charset utf-8;
+
+access_log /var/log/nginx/openhabcloud.org-access.log;
+error_log /var/log/nginx/openhabcloud-error.log;
+client_max_body_size 300m;
+
+location /css {
+    alias  /home/ubuntu/openhabcloud/public/css;
+    }
+location /js {
+    alias /home/ubuntu/openhabcloud/public/js;
+    }
+location /img {
+    alias /home/ubuntu/openhabcloud/public/img;
+    }
+location /bootstrap {
+    alias /home/ubuntu/openhabcloud/public/bootstrap;
+    }
+location /font-icons {
+    alias /home/ubuntu/openhabcloud/public/font-icons;
+    }
+location /fonts {
+    alias /home/ubuntu/openhabcloud/public/fonts;
+    }
+location /js-plugin {
+    alias /home/ubuntu/openhabcloud/public/js-plugin;
+    }
+location /downloads {
+    alias /home/ubuntu/openhabcloud/public/downloads;
+    }
+location / {
+    proxy_pass http://localhost:3000;
+    proxy_redirect off;
+    proxy_http_version 1.1;
+    proxy_set_header Host $host;
+    proxy_set_header Upgrade $http_upgrade;
+    proxy_set_header Connection "upgrade";
+    proxy_set_header X-Real-IP $remote_addr ;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for ;
+        proxy_set_header X-Forwarded-Proto https;
+}
+
+#error_page 404 /404.html;
+
+# redirect server error pages to the static page /50x.html
+#
+error_page 500 502 503 504 /50x.html;
+location = /50x.html {
+root html;
+}
+}
+```
+
+
+You need to restart nginx:
+
+```
+sudo service nginx restart
+```
+
+
+Now we are ready to start the openHAB Cloud service.
+
+We will run openHAB Cloud as background service:
+```
+sudo nohup node app.js &
```

Comment:
this is not specific to EC2, i would remove it.  Also there is etc/openhabcloud.service which is a systemd startup script that can be used to start and stop the cloud service on Ubuntu.